pkg/kapp/resources/resources.go

```diff
@@ -381,7 +381,7 @@ func (c *ResourcesImpl) Exists(resource Resource) (bool, error) {
 	var found bool
```

Comment:
instead of modifying existing behavious of Exists() (since it might be used in different places that do not care about UID), i was thinking should we add ExistsOpts{SameUID: true} optional arg?